crate_universe/src/lockfile.rs

```diff
@@ -211,7 +211,7 @@ mod test {
 
         assert_eq!(
             digest,
-            Digest("7be4f323ac6a4d0a45d9d430a8056967eb248ca7c86bdba44af33ad90392cb4a".to_owned())
+            Digest("7b4142c24aff865f158da49840f7f23a6921c65f28999dd03fdd4bc83eb3fcb1".to_owned())
```

Comment:
With the updated implementation for `RenderConfig::default()` this hash needs to be updated.